src/System.Transactions/tests/CloneTxTests.cs

```diff
@@ -115,9 +124,34 @@ public void Run(CloneType cloneType, IsolationLevel isoLevel, bool forcePromote,
             {
                 tx.Commit();
             }
-            catch (TransactionAbortedException)
+            catch (TransactionAbortedException ex)
             {
                 Assert.Equal(expectedStatus, TransactionStatus.Aborted);
+                switch (cloneType)
+                {
+                    case CloneType.Normal:
+                        {
+                            // We shouldn't be getting TransactionAbortedException for "normal" clones,
+                            // so we have these two Asserts to possibly help determine what went wrong.
+                            Assert.Null(ex.InnerException);
+                            Assert.Equal(ex.Message, "There shouldn't be any exception with this Message property");
```

Comment:
Nit: the first parameter to Equal is expected, the second is actual, and the order matters in terms of the error message that's generated if the assertion fails.  Looks like these args should be swapped.  Same goes for other cases of this in your changes.